doc/guide/Guide_Installation.ipynb

```diff
@@ -246,7 +256,15 @@
     "  from climada.util import log_level\n",
     "  with log_level(level='WARNING'):\n",
     "        quietly_do_something()\n",
-    "  ```\n"
+    "  ```\n",
+    "\n",
+    "* **How to install climada_petals in a separate environment**\\\n",
+    "  If you want to have a separate environment for CLIMADA-petals, procede as described above but create an environment instead of updating:\n",
```

Comment:
```suggestion
    "  If you want to have a separate environment for CLIMADA-petals, proceed as described above but create an environment instead of updating:\n",
```